{"name":"User repository update user failure","fullName":"TestUserUpdateSuite/User repository update user/TestUpdate_Failure","status":"passed","statusDetails":{"message":"","trace":""},"start":1727773563257,"stop":1727773563257,"uuid":"626857ac-7fd4-11ef-a987-ca15b1947338","historyId":"2a66bdc0b59b55f4fb043b08c4ecb884","testCaseId":"0481dc0ad7e887a4fa8c385de6a10b9d","labels":[{"name":"language","value":"go1.22.5"},{"name":"framework","value":"Allure-Go@v0.6.0"},{"name":"host","value":"h19.net58.bmstu.ru"},{"name":"thread","value":"TestUserUpdateSuite/User repository update user/TestUpdate_Failure"},{"name":"suite","value":"User repository update user"},{"name":"package","value":"github.com/paw1a/eschool/internal/adapter/repository/postgres/test"}],"steps":[{"name":"ASSERT: Error Is","status":"passed","start":1727773563257,"stop":1727773563257,"parameters":[{"name":"Error","value":"sql: connection is already closed: record update failed"},{"name":"Target","value":"record update failed"}]}]}